src/xrScriptEngine/script_engine.cpp

```diff
@@ -754,8 +753,20 @@ void CScriptEngine::lua_error(lua_State* L)
 
 int CScriptEngine::lua_pcall_failed(lua_State* L)
 {
-    print_output(L, "", LUA_ERRRUN);
+#if (!defined(DEBUG) && !XRAY_EXCEPTIONS)
+    print_output(L, "", 0); // Force game to not break in raii_guard() for this type of errors
+#else
+    print_output(L, "", LUA_ERRRUN); // Default behavior
+#endif
```

Comment:
@Armada651 That's how it is originally designed (it has to be fixed, but that's another story). So, are we good now?